Pinta.Effects/Effects/RedEyeRemoveEffect.cs

```diff
@@ -75,7 +75,7 @@ public class RedEyeRemoveData : EffectData
 		
 		[MinimumValue (0), MaximumValue (100)]
 		[Caption ("Saturation percentage")]
-		[Hint ("Hint: For best results, first use selection tools to select each eye.")]
+		[Hint ("Hint: For best results, first use \nselection tools to select each eye.")]
```

Comment:
I think the correct fix is to adjust SimpleEffectDialog so that the labels can wrap around?